spec/controllers/admin/users_controller_spec.rb

```diff
@@ -19,4 +19,24 @@
     end
   end
 
+  describe '#lock_account' do
+    it 'it locks the given account' do
+      other_user = FactoryGirl.create :user
+      expect(other_user).to receive(:lock_access!)
+      allow(User).to receive(:find).and_return(other_user)
+
+      post other_user.expect(other_user.reload.access_locked?).to be_truthy
```

Comment:
Yes it is. `expect` is not monkey patched onto all objects, it's only available inside the `it` block. Just drop the `other_user.` before it, I'm not following what that should express anyway ;)